assembl/static2/js/app/pages/signup.jsx

```diff
@@ -1,22 +1,37 @@
 import React from 'react';
 import { Grid, Row, Col } from 'react-bootstrap';
+import { connect } from 'react-redux';
 import SignupForm from '../components/login/signupForm';
 import SignUpConfirm from '../components/login/signUpConfirm';
 
 class Signup extends React.Component {
+
   render() {
-    const isAccountCreated = location.hash === '#success';
+    // console.log(this.props);
+    let signupConfirm = false;
+    // Data has been fetched from the backend
+    if ('signupSuccess' in this.props.auth) {
+      if (this.props.auth.signupSuccess.success === true) {
+        signupConfirm = this.props.auth.signupSuccess.success;
+      }
+    }
     return (
       <Grid fluid className="login-container">
         <Row className="max-container center">
           <Col xs={12} md={6} className="col-centered">
-            {!isAccountCreated && <SignupForm />}
-            {isAccountCreated && <SignUpConfirm />}
+            {!signupConfirm && <SignupForm />}
+            {signupConfirm && <SignUpConfirm />}
```

Comment:
Haha, fair enough. I've made a fair bit of comments on #90 to use this style as well, since it appears both you and @paolina92 like this approach more.